rotkehlchen/exchanges/data_structures.py

```diff
@@ -534,6 +547,18 @@ class Loan(AccountingEventMixin):
     def get_timestamp(self) -> Timestamp:
         return self.close_time
 
+    def serialize(self) -> Dict[str, Any]:
+        """Serialize the loan into a dict."""
+        return {
+            'location': self.location.serialize(),
+            'open_time': self.open_time,
+            'close_time': self.close_time,
+            'currency': self.currency.serialize(),
```

Comment:
```suggestion
            'currency': self.currency.identifier
```

Reply:
`serialize()` calls `.identifier` or is there any specific to do this?

Reply:
because that's how we do it everywhere. Why have one extra call? I do understand that if we at some point want to change the serialization of `Assets` things would be easier if we have it abstracted in a function but ... hm ... yeah that would already need many changes all over the codebase.